bijection-hbase/src/test/scala/com/twitter/bijection/hbase/HBaseInjectionsLaws.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.twitter.bijection.hbase
+
+import org.scalatest.{ PropSpec, MustMatchers }
+import org.scalatest.prop.PropertyChecks
+import org.scalacheck.Arbitrary
+
+import com.twitter.bijection.{ CheckProperties, BaseProperties, Injection }
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable
+
+object HBaseInjectionsLaws {
+  implicit val arbitaryImmutableBytesWritable: Arbitrary[ImmutableBytesWritable] =
```

Comment:
this is not robust enough. I think we need something like this:
```scala
for {
  bytes <- Arbitrary.arbitrary[Array[Byte]]
  offset <- Gen.choose(0, bytes.length - 1)
  len <- Gen.choose(0, bytes.length - offset)
} yield new ImmutableBytesWritable(bytes, offset, len)
```
so we can exercise having something that has offset/length other than the trivial values.